src/pages/signatures/graphql/examples.mdx

```diff
@@ -72,6 +72,20 @@ import displayDocumentIDExample from '../../../examples/createSignatureOrder/dis
 
 <GraphQLExample example={displayDocumentIDExample} />
 
+### Custom seals page template
+
+By default, a blank page is appended to the signed document to display the signature seals. You can override this behavior by providing a template for the seals page. This allows you to add custom branding, such as logos and brand colors, to the page where the seals are placed.
```

Comment:
"This feature is a paid add-on", or something to that effect

Reply:
Added in [52e0561](https://github.com/criipto/docs/pull/202/commits/52e05614d5a20a6a19558e3c47f352cdbbb12bd9)